### PR TITLE
Remove LSO steps from assisted-operator workflow

### PR DESCRIPTION
**Pull request by YuviGold**
The entire disconnected logic should be implemented in the same place
as where the connected logic is located.
Currently that's assisted-service repository hence moving the logic
from openshift/release including deleting LSO steps.

assisted-service PR - https://github.com/openshift/assisted-service/pull/2052

@osherdp @lranjbar 
/hold

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh [hunk 1]
```diff
@@ -36,44 +37,11 @@ cd "\${REPO_DIR}"
...
-      }
-    )')
-
-if [ "\${DISCONNECTED}" = "true" ]; then
-  export MIRROR_BASE_URL="http://\$(wrap_if_ipv6 \${PROVISIONING_HOST_IP})/images"
-  export IRONIC_IMAGES_DIR
-fi
-
 images=(${ASSISTED_AGENT_IMAGE} ${ASSISTED_CONTROLLER_IMAGE} ${ASSISTED_INSTALLER_IMAGE})
 export PUBLIC_CONTAINER_REGISTRIES=\$(for image in \${images}; do echo \${image} | cut -d'/' -f1; done | sort -u | paste -sd "," -)
```
Comment (osherdp) [inline ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh:47]: hmm, you meant to call ``deploy/operator/deploy.sh``?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by YuviGold** [comment]
https://github.com/openshift/assisted-service/pull/2086 should fix semi-colon issue.

**Comment by YuviGold** [comment]
PR is passing for ocm-2.3 and master branches.
Removing the patch to trigger them.

**Review by osherdp — APPROVED** [review]

**Pull request merged**